packages/rollup-plugin-polyfills-loader/rollup-plugin-polyfills-loader.js

```diff
@@ -72,7 +73,12 @@ function rollupPluginPolyfillsLoader(pluginOptions = {}) {
         let preloaded = [];
         for (const entrypoint of entrypoints) {
           preloaded.push(entrypoint.importPath);
-          preloaded.push(...entrypoint.chunk.imports);
+
+          // js files (incl. chunks) will always be in the root directory
+          const pathToRoot = path.relative('./', path.dirname(entrypoint.importPath));
```

Comment:
```suggestion
          const pathToRoot = path.posix.relative('./', path.posix.dirname(entrypoint.importPath));
```

Reply:
I don't understand what this does. It create a path relative to the same directory?

Reply:
jup you get `foo/index.html` and it will give you `../`  if you have `index.html` it gives you ``

Reply:
This is what I get in the node terminal>

```js
const path = require('path');
> path.relative('./', 'foo/bar')
'foo/bar'
> path.relative('./', 'foo')
'foo'
```

Reply:
```
{
  importPath: './inline-module-index-1.js',
  pathToRoot: '',
  preload: 'navigation.js'
}
{
  importPath: '../inline-module-index-12.js',
  pathToRoot: '..',
  preload: '../navigation.js'
}
```

so we "extract" the leading ".." so we can add them to our chunk path which will then be the preload path

Reply:
Ah now I understand, makes sense. That would automatically work when files are output into directories as well. So we can remove the comment and the point below is solved as well.